src/jobs/release.yml

```diff
@@ -0,0 +1,57 @@
+description: >
+  Create a new GitHub Release for a repository.
+  Specify a tag with the `tag` parameter (required). If a matching git tag does not yet exist, one will automatically get created from the latest state of the default branch.
+
+docker:
+  - image: cimg/base:stable
+
+resource_class: small
+
+parameters:
+  tag:
+    type: string
+    description: "Select a tag to create a release from. If a matching git tag does not yet exist, one will automatically get created from the latest state of the default branch."
+  notes-file:
+    description: "Path to file containing release notes (e.g.: ./changelog.md). "
+    default: ""
+    type: string
+  draft:
+    type: boolean
+    default: false
+    description: Save the release as a draft instead of publishing it.
+  prerelease:
+    type: boolean
+    default: false
+    description: Mark the release as a prerelease.
+  files:
+    description: "To include a subset of your repository's files in the release, enter the filt glob here. (e.g. ./dist/*.tgz) "
+    default: ""
+    type: string
+  title:
+    description: "Include a release title. "
+    type: string
+    default: ""
+  version:
+    type: string
+    default: "1.9.2"
+    description: Specify the full semver versioned tag to use for the GitHub CLI installation.
+  token:
+    type: env_var_name
+    default: "GITHUB_TOKEN"
+    description: |
+      Enter the name of the environment variable containing the GitHub Personal Access token to be used for authentication.
+      It is recommended for CI processes that you create a "machine" user on GitHub.com with the needed permissions, rather than using your own.
+steps:
+  - setup:
+      version: <<parameters.version>>
+      token: <<parameters.token>>
+  - run:
+      name: "Creating a <<#parameters.draft>><<parameters.draft>> <</parameters.draft>>GitHub Release"
```

Comment:
Wonder if this might be clearer as `<<#parameters.draft>>draft <</parameters.draft>>`  instead?